atd-vze/src/views/Crashes/personDataMap.js

```diff
@@ -108,7 +108,7 @@ export const primaryPersonDataMap = [
         editable: false,
       },
       peh_fl: {
-        label: "Unhoused",
+        label: "Suspected Unhoused",
```

Comment:
is there a preference on sentence case vs title case @patrickm02L ?

Reply:
All the other labels use title case so i went with that

Reply:
i just noticed that when i was testing it and was going to remove my comment!